examples/boutcore/data/BOUT.inp

```diff
@@ -1,16 +1,16 @@
 nout = 10
 timestep = 0.1
 
-mxg = 1
-myg = 1
+MXG = 1
+MYG = 1
```

Comment:
Can these top-level options be removed? There are MXG, MYG settings in the `mesh` section too

Reply:
One for @dschwoerer ?

Reply:
If the top-level options are removed, the mesh section ones are used, and the simulation fails.

I'll push a fix.

Reply:
Lovely, thanks @dschwoerer :+1: